src/Fantomas/Context.fs

```diff
@@ -108,63 +115,67 @@ type internal Context =
             Positions = positions 
             Trivia = trivia }
 
-    member x.MemoizeProjection = x.Writer.Column, x.Trivia, x.BreakLines, x.RecordBraceStart
+    member x.MemoizeProjection = x.WriterInitModel, x.WriterEvents, x.Trivia, x.BreakLines, x.RecordBraceStart
     
-    member x.With(writer : ColumnIndentedTextWriter, ?keepPageWidth) =
+    member x.With(writerCommands, ?isDummy, ?keepPageWidth) =
```

Comment:
Didn't find any occurrence where `isDummy` is false so maybe drop the parameter and rename to `WithDummy`?

Reply:
Great idea! Will do.